content/flow-collections/exercises/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "Exercises"
+date: 2023-01-25T13:44:27-06:00
+draft: false
+weight: 7
+originalAuthor: <no value> # to be set by page creator
+originalAuthorGitHub: <no value> # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
```

Comment:
Please fix the Check your solution formatting, make sure to tab in under the ordered list.

Reply:
done